src/airflow_tools/providers/http_to_data_lake/operators/http_to_data_lake.py

```diff
@@ -190,6 +196,20 @@ def _response_filter(self, response) -> BytesIO:
                     )
                 return list_to_jsonl(self.data, self.compression)
 
+            case 'xml':
+                if not isinstance(self.data, str):
+                    raise ApiResponseTypeError(
+                        'Expected response can\'t be transformed to xml. It is not a string'
+                    )
+                return xml_to_binary(self.data, self.compression)
+
+            case 'parquet':
+                if not isinstance(self.data, str):
+                    raise ApiResponseTypeError(
+                        'Expected response can\'t be transformed to parquet. It is not a string'
```

Comment:
Same here

Reply:
I think if we change the initial if statements this should not be needed.